### PR TITLE
[Bug] Fix non-lazy ROIExtractors import

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Failed to notice this with PR #731, my bad. Imports like this should be lazy and done within the `__init__` of the class so the outer imports don't fail under a minimal installation

### CODE REVIEW

File: requirements-testing.txt [hunk 1]
```diff
@@ -4,4 +4,3 @@ ndx-events>=0.2.0  # for special tests to ensure load_namespaces is set to allow
 parameterized>=0.8.1
 ndx-miniscope
 spikeinterface[qualitymetrics]>=0.99.1
```
Comment (h-mayorquin) [inline requirements-testing.txt:6]: why do we still have spikeinterface though?

**Review by h-mayorquin — APPROVED** [review]
Seems good. You are checking that the you can import interfaces but that they fail at run time, right?

File: requirements-testing.txt (hunk 1 above)
Comment (CodyCBakerPhD) [reply requirements-testing.txt:6]: Not SpikeInterface in particular, but the deps it needs for the `[qualitymetrics]` extra are used to test an advance feature of SI (sparisty I believe? Or something like that)

**Comment by CodyCBakerPhD** [comment]
> You are checking that the you can import interfaces but that they fail at run time, right?

In the particular case of minimal install in fresh environment, yess

**Pull request merged**